tests/name_tests.rs

```diff
@@ -39,10 +39,7 @@ fn test_dns_name_traits() {
     compile_time_assert_sync::<DnsName>();
 
     let a_ref = DnsNameRef::try_from_ascii(b"example.com").unwrap();
-
-    // `From<DnsNameRef>`
-    // TODO(XXX): Remove when deprecated From<DnsNameRef> for DnsName trait is removed.
-    let a: DnsName = DnsName::from(a_ref);
+    let a: DnsName = a_ref.to_owned();
```

Comment:
Style nit: would be nice to lose the type annotation here, which I think should work?

Reply:
Fixed, thanks!